### ISSUE TITLE
strange characters in torrent names for foreign language

### ISSUE THREAD

**Issue by theothersophie** [kz26/dottorrent-gui#42]
I have a bunch of files in french and all the accented characters get replaced by some strange characters, for example I get a torrent named "R├⌐fl├⌐chissez" from a file named "Réfléchissez"

**Comment by wayward-turnip** [comment]
Correct, dottorrent fails to work if any of the filenames contain accented characters (å, ñ, etc.).   This is a nasty bug.
